src/PowerShellEditorServices/Session/RunspaceDetails.cs

```diff
@@ -218,7 +218,16 @@ internal bool HasCapability<TCapability>()
                     // ProcessId property isn't on the object, move on.
                 }
 
-                if (runspace.ConnectionInfo.ComputerName != "localhost")
+                // Grab the $host.name which will tell us if we're in a PSRP session or not
+                string hostname =
+                        PowerShellContext.ExecuteScriptAndGetItem<string>(
+                            "$host.name",
+                            runspace,
+                            defaultValue: "");
```

Comment:
```suggestion
                            defaultValue: string.Empty);
```

Reply:
something in my mind said this wasn't supported in Framework... maybe I was thinking of Array.Empty or whatever it is.

Reply:
Yeah that one isn't 😢